src/Buttons.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import VTMap from "./VTMap";
+import leafletPip from "@mapbox/leaflet-pip";
+import borderData from "./border";
+import L from "leaflet";
+import { Map } from "react-leaflet";
+
+class Buttons extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      disabled: true,
+      gameStarted: false,
+      long: "",
+      lat: "",
+      zoom: 7,
+      marker: {},
+      modalOpen: false,
+    };
+  }
+
+  //---------Handles click event-------------//
+  handleClick = (event) => {
+    this.setState({
+      gameStarted: true,
+    });
+  };
+
+  //---------Handles event after guess button is clicked-------------//
+  handleGuessButton = (event) => {
+    event.preventDefault();
+
+    this.setState({
+      gameStarted: true
+    });
+  };
+
+  //---------Handles event after quit button is clicked-------------//
+  handleQuitButton = (event) => {
+    event.preventDefault();
+
+    this.setState({
+      gameStarted: false,
+      zoom: 7,
+      modalOpen: true,
+    });
+  };
+
+handleModalOpen = () => {
+
+}
```

Comment:
The way you are currently doing it (passing methods as props to children to update the parent's state) is the correct way to go about this. You just don't need these methods here, and you should delete unused code to keep your files clean, and organized.